Add codebytes to Javascripts statements entry #4155 (#4186)
Codecademy/docs: content/javascript/concepts/statements/statements.md

> **Note:** JavaScript identifiers are case-sensitive. Notice that `x` is not the same as `X`.

> ## codebyte Example

Reviewer: ```suggestion
## Codebyte Example
```

> ## codebyte Example

Reviewer 2: ```suggestion
## Codebyte Example
```

> This example shows the output values of the declared variables

Reviewer: ```suggestion
This example shows the output values of the declared variables:
```

> ```codebyte/js

Reviewer 2: ```suggestion
```codebyte/javascript
```
Correct this for every codebyte.

> ```codebyte/javascript

Reviewer: ```suggestion
```codebyte/js
```

Author: Hi @dakshdeepHERE. Thanks a lot for your review and suggestions. I earnestly appreciate them. I am currently making the necessary changes. Please kindly confirm that this entry means that I should change all ````codebyte/javascript to codebyte/js.

var x = 5;
console.log(x)
// outputs 5
var y = 6;
console.log(y)
// outputs 6
var x = 4;
console.log(x)
// outputs 4
var z = x + y;
console.log(z)

> // outputs 10, x is reassigned 4

Reviewer: remove the comments with the output. There's no need for these comments the readers can compile the code themselves.

> **Note:** JavaScript identifiers are case-sensitive. Notice that `x` is not the same as `X`.

## Codebyte Example

This example shows the output values of the declared variables:

```codebyte/js
var x = 5;
console.log(x)

var y = 6;
console.log(y)

var x = 4;
console.log(x)

var z = x + y;
console.log(z)

> 

Reviewer 2: ```suggestion
```

// Function call
console.log('Hello World');
```

> ## Codebyte Example

Reviewer 2: ```suggestion

## Codebyte Example
```

This example shows a `console.log()` function call:

> ```codebyte/js

Reviewer 2: ```suggestion
```codebyte/javascript
```

// Function call
console.log('Hello World');
```

> ## codebyte Example

Reviewer: ```suggestion
## Codebyte Example
```

> This example shows a `console.log()`  function call

Reviewer: ```suggestion
This example shows a `console.log()` function call:
```

```codebyte/javascript
console.log('Hello, World!');

> // outputs Hello, World!

Reviewer: ```suggestion
```

console.log(x);
```

## Codebyte Example

> This example shows re-assignment of value to variable `x`:

Reviewer 2: ```suggestion
This example shows the re-assignment of value to variable `x`:
```

> ```codebyte/js

Reviewer 2: ```suggestion
```codebyte/javascript
```

var x = 5;
console.log(x)
x = 6;
console.log(x);
```

> 

Reviewer 2: ```suggestion
```

console.log(x);
```

> ## codebyte Example

Reviewer: ```suggestion
## Codebyte Example
```

> This example shows re-assignment of value to variable `x`

Reviewer: ```suggestion
This example shows the re-assignment of value to variable `x`:
```

> ```codebyte/javascript

Reviewer: ```suggestion
```codebyte/js
```

> var x = 5;
> console.log(x)
> //outputs 5
> x = 6;
> console.log(x);
> // outputs 6

Reviewer: ```suggestion
var x = 5;
console.log(x)
x = 6;
console.log(x);
```

if (x > 5) {
  console.log('x is greater than 5');
}
```

> ## Codebyte Example

Reviewer 2: ```suggestion

## Codebyte Example
```

This example shows the output of an if statement:

> ```codebyte/js

Reviewer 2: ```suggestion
```codebyte/javascript
```

if (x > 5) {
  console.log('x is greater than 5');
}
```

> ## codebyte Example

Reviewer: ```suggestion

## Codebyte Example
```

> This example shows the output of an if statement

Reviewer: ```suggestion
This example shows the output of an if statement:
```

> ```codebyte/javascript

Reviewer: ```suggestion
```codebyte/js
```

var x = 7
if (x > 5) {
  console.log('x is greater than 5');

>   // outputs x is greater than 5

Reviewer: ```suggestion
```

    console.log('x is neither 1 nor 2');
}
```

> This example shows outputs of switch statement:

Reviewer 2: ```suggestion

This example shows the outputs of the switch statement:
```

> ```codebyte/js

Reviewer 2: ```suggestion
```codebyte/javascript
```

// change value of x 
var x = 1; 

switch (x) {
  case 1:
    console.log('x is 1');
    break;
  case 2:
    console.log('x is 2');
    break;
  default:
    console.log('x is neither 1 nor 2');
}
```

> Change the values of x to different numbers and click on run button to see their outputs.

Reviewer 2: ```suggestion

Change the values of x to different numbers and click on the run button to see their outputs.
```
Give a line space before and after a code block.

    console.log('x is neither 1 nor 2');
}
```

> This example shows outputs of switch statement:

Reviewer 2: ```suggestion

This example shows the outputs of the switch statement:
```

    console.log('x is neither 1 nor 2');
}
```

> This example shows outputs of switch statment

Reviewer: ```suggestion
This example shows outputs of switch statement:
```

  console.log(x);
  x++;
}
```
## Codebyte example

> ```codebyte/js

Reviewer 2: ```suggestion
```codebyte/javascript
```

// While loop
var x = 0;
while (x < 7) {
  console.log(x);
  x++;
}
```

```js
// For loop
for (let i = 0; i < 5; i++) {
  console.log(i);
}
```
## Codebyte example

> This works similarly to the `while`  loop. Conventionally `let`  keyword is used to declare a variable named `i`  in `for`  loops:

Reviewer 2: ```suggestion
This works similarly to the `while`  loop. Conventionally `let` keyword is used to declare a variable named `i`  in `for`  loops:
```

> ```codebyte/js

Reviewer 2: ```suggestion
```codebyte/javascript
```

  console.log(x);
  x++;
}
```

> ## codebyte example

Reviewer: ```suggestion

## Codebyte example
```

```codebyte/javascript
// While loop
var x = 0;
while (x < 7) {
  console.log(x);
  x++;
}
```

```js
// For loop
for (let i = 0; i < 5; i++) {
  console.log(i);
}
```

> ## codebyte example

Reviewer: ```suggestion

## Codebyte example
```

> This works similar to `while`  loop. Conventionally `let`  keyword is used to declare a variable named `i`  in `for`  loops

Reviewer: ```suggestion
This works similarly to the `while`  loop. Conventionally `let`  keyword is used to declare a variable named `i`  in `for`  loops:
```

  }
  console.log(i);
}
```
## Codebyte Example

This code breaks or exits the loop when  `i` equals 3 and returns the value of `i`:

> ```codebyte/js

Reviewer 2: ```suggestion
```codebyte/javascript
```

// Break statement
for (let i = 0; i < 5; i++) {
  if (i === 3) {
    break;
  }
  console.log(i);
}

```

```js
// Return statement
function add(x, y) {
  return x + y;
}
```
## Codebyte Example

This code example returns the value of the add function: 

> ```codebyte/js

Reviewer 2: ```suggestion
```codebyte/javascript
```

function add(x, y) {
  return x + y;
}
console.log(add(2, 4))

> 

Reviewer 2: ```suggestion
```

  }
  console.log(i);
}
```

> ## codebyte Example

Reviewer: ```suggestion

## Codebyte Example
```

> This code breaks or exits the loop when  `i` equals 3 and returns the value of `i`

Reviewer: ```suggestion
This code breaks or exits the loop when  `i` equals 3 and returns the value of `i`:
```

```codebyte/javascript
// Break statement
for (let i = 0; i < 5; i++) {
  if (i === 3) {
    break;
  }
  console.log(i);
}

```

```js
// Return statement
function add(x, y) {
  return x + y;
}
```

> ## codebyte Example

Reviewer: ```suggestion

## Codebyte Example
```

> This code example returns the value of the add function 

Reviewer: ```suggestion
This code example returns the value of the add `function`:
```

```codebyte/javascript
function add(x, y) {
  return x + y;
}

> // call the add function with values of 2 and 4 for the arguements x and y and print the result to the console

Reviewer: ```suggestion
```

  }
  console.log(i);
}
```

## Codebyte Example

This code breaks or exits the loop when  `i` equals 3 and returns the value of `i`:

```codebyte/javascript
// Break statement
for (let i = 0; i < 5; i++) {
  if (i === 3) {
    break;
  }
  console.log(i);
}
```

```js
// Return statement
function add(x, y) {
  return x + y;
}
```

## Codebyte Example

This code example returns the value of the add function: 

> ```codebyte/javascript

Reviewer 2: ```suggestion
```codebyte/js
```
